eng/scripts/Language-Settings.ps1

```diff
@@ -89,16 +89,33 @@ function Get-dotnet-PackageInfoFromPackageFile ($pkg, $workingDirectory)
   }
 }
 
+# Return list of nupkg artifacts
+function Get-dotnet-Package-Artifacts ($Location)
+{
+  $pkgs = Get-ChildItem "${Location}" -Recurse | Where-Object -FilterScript {$_.Name.EndsWith(".nupkg") -and -not $_.Name.EndsWith(".symbols.nupkg")}
+  if (!$pkgs)
+  {
+    Write-Host "$($Location) does not have any package"
+    exit(1)
+  }
+  elseif ($pkgs.Count -ne 1)
+  {
+    Write-Host "$($Location) should contain only one (1) published package"
+    Write-Host "No of Packages $($pkgs.Count)"
+    exit(1)
+  }
+  return $pkgs[0]
+}
+
 # Stage and Upload Docs to blob Storage
 function Publish-dotnet-GithubIODocs ($DocLocation, $PublicArtifactLocation)
 {
-  $PublishedPkgs = Get-ChildItem "$($DocLocation)" | Where-Object -FilterScript {$_.Name.EndsWith(".nupkg") -and -not $_.Name.EndsWith(".symbols.nupkg")}
-  $PublishedDocs = Get-ChildItem "$($DocLocation)" | Where-Object -FilterScript {$_.Name.EndsWith("docs.zip")}
+  $PublishedPkg = Get-dotnet-Package-Artifacts $DocLocation
+  $PublishedDocs = Get-ChildItem "${DocLocation}" | Where-Object -FilterScript {$_.Name.EndsWith("docs.zip")}
 
-  if (($PublishedPkgs.Count -gt 1) -or ($PublishedDoc.Count -gt 1))
+  if ($PublishedDoc.Count -gt 1)
   {
       Write-Host "$($DocLocation) should contain only one (1) published package and docs"
-      Write-Host "No of Packages $($PublishedPkgs.Count)"
       Write-Host "No of Docs $($PublishedDoc.Count)"
       exit 1
```

Comment:
sigh... I see we are doing an exit here as well. We will have likely want to clean these up at some point.

Reply:
I will check this and clean up in separate PR in languages.